spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java

```diff
@@ -278,6 +275,13 @@ private static Integer obtainManagementServerPort(ServerBuilder serverBuilder,
         }
 
         try {
+            final boolean actuatorEnabled =
+                    (boolean) ACTUATOR_ENABLED_METHOD.invoke(internalServices);
+            final Port internalPort = (Port) INTERNAL_SERVER_PORT_METHOD.invoke(internalServices);
+            if (actuatorEnabled && internalPort != null) {
+                return internalPort.getPort();
+            }
+
             final Port port = (Port) MANAGEMENT_SERVER_PORT_METHOD.invoke(internalServices);
```

Comment:
It seems like the actuator is not bound to the management server port if the internal service port is configured.
Shouldn't we bind it to both?

Reply:
That's right.
So if the internal service port is configured and the actuator is specified, `WebOperationService` is bound into the internal port around here.
https://github.com/line/armeria/blob/a1105519e920b00ee3bc848a5074d943925931ef/spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java#L218

Also `WebOperationService` is bound into the management server port if the server has it ?

Reply:
> Also WebOperationService is bound into the management server port if the server has it ?

I believe that it's the expected behavior. 😉